api/management/commands/create_groups.py

```diff
@@ -17,15 +22,21 @@ def add_arguments(self, parser: CommandParser) -> None:
         parser.add_argument("--teacher_ids", nargs="*", type=int)
 
     def handle(self, *_: str, **options: Any) -> None:
-        teachers: Iterable[Teacher] = []
+        teachers: Iterable[Teacher]
         if options["teacher_ids"]:
-            teachers = Teacher.objects.filter(pk__in=options["teacher_ids"])
-            for teacher in teachers:
+            teachers = []
```

Comment:
the variable has the same name as the one above. maybe `teachers_for_group_candidate`?

Reply:
this may be irrelevant